pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -352,6 +358,7 @@ pub mod pallet {
 					currency_ids,
 					transferable,
 					denomination,
+					min_operation_balance,
```

Comment:
should we force this to be at least 1 (e.g., simply replace if 0)? It makes no sense anyway to try to mint or burn 0 coins...

Reply:
Throwing an error because of this seems harsh, and 0 and 1 are identical in effect - minting / burning 0 will already always fail (due to ZeroCollateral), so silently raising a 0 to a 1 minimum just means throwing a different error instead (BelowMinimum). Do we need to bug people just trying to create a pool with default parameters with this detail?